lua/modules/linear_bolzmann_solvers/lbs_solver/lbs_sources.cc

```diff
@@ -55,16 +55,16 @@ RegisterLuaFunctionInNamespace(AddDistributedSource, lbs, AddDistributedSource);
 int
 AddDistributedSource(lua_State* L)
 {
-  const std::string fname = "lbs.AddDistributedSource";
+  const std::string fname = "lbs.AddVolumetricSource";
```

Comment:
Volumetric implies space-dependent. The source is also already group-dependent. Eventually, it would be good to have the source be direction-dependent. So, given this, wasn't distributed better than volumetric? maybe there is something better than distributed.

Reply:
I axed distributed because calling something "distributed" in a parallel code to avoid any confusion between physics and computer science language. Volumetric does imply space-dependent, and this is a space-dependent source. I agree that it would be good to have a direction dependent source. I am open to proposals for names, but I think in most cases if you say volumetric source, it will be understood as the right-hand side `Q` in the transport equation.

Reply:
Comment. Ok, volumetric is clear. I think it is already group dependent. Do you think MG_volumetric could make it even more obvious? (Any version that includes mg or multigroup may have my slight preference)

Reply:
At this point, I think that it is okay to leave off multigroup. The old source `IsotropicMultiGroupSource` was housed in `framework`, making the extra clarification more necessary. Because `VolumetricSource`, like `PointSource`, is housed within the LBS module, in my mind multigroup is essentially implied. For example, `PointSource` is not `MultiGroupPointSource`. Eventually, if we would like to extend this to be arbitrary in angle as well, it may be worthwhile to rename `VolumetricSource` -> `IsotropicVolumetricSource` and `PointSource` -> `IsotropicPointSource` to make room for the generic `VolumetricSource` and `PointSource` classes that are arbitrary in angle.

Reply:
I agree on leaving out MG since this is housed in `lbs`. I like the addition of Isotropic but because this needs to happen for both volumetic and point sources, it could be preferable to do that renaming in a new PR